tests/scenarios/package.json

```diff
@@ -4,6 +4,7 @@
   "dependencies": {
     "@embroider/shared-internals": "1.5.0",
     "@types/qunit": "^2.11.1",
+    "execa": "^5.0.0",
```

Comment:
Should this be a devDep?

Reply:
yes! thanks

Reply:
just kidding, this is required to be a dep, due to lints